### ISSUE TITLE
Alternating weeks

### ISSUE THREAD

**Issue by idkwhatyouwant** [CoBrA2168/Sleeper#4]
Would it be possible to add the option to have an alarm that is on/skipped every X week? I work a 9/80 schedule that gives me every other Friday off. I purchased this tweak since it can achieve this function if I manually enter the dates off, but it is fairly tedious to do for multiple alarms. Thanks!

**Comment by joshuaseltzer** [comment]
Yes this should very much be possible. Maybe I could sneak it out as a minor update before doing the next major release (which will likely be iOS 15 support).

**Comment by idkwhatyouwant** [comment]
Thank you so much, I’m looking forward to it!
